monai/transforms/croppad/dictionary.py

```diff
@@ -265,11 +265,13 @@ class RandCropByPosNegLabeld(Randomizable, MapTransform):
     Args:
         keys (list): parameter will be used to get and set the actual data item to transform.
         label_key: name of key for label image, this will be used for finding foreground/background.
-        size (list, tuple): the size of the crop region e.g. [224,224,128]
+        spatial_size (sequence of int): the spatial size of the crop region e.g. [224, 224, 128]
         pos: used to calculate the ratio ``pos / (pos + neg)`` for the probability to pick a
-          foreground voxel as a center rather than a background voxel.
+            foreground voxel as a center rather than a background voxel. if both pos and neg are None,
+            use a randomly selected postion in the image as crop center.
```

Comment:
Hi @wyli ,

This feature is to generate random samples instead of just 1 cropped image, for example, if we have 10 images in the dataset and want to use batch size > 10.
I agree with you to change this logic to `RandSpatialCrop` or a new `RandSpatialCropSamples`, let me update this PR soon.
And I plan to add Array version RandPosNegCrop and refactor all the logic soon, it will make the it more clear.
Thanks.

Reply:
thanks, a new `RandSpatialCropSamples` sounds great. could have the array version after 0.2.0

Reply:
updated in the latest commit.